src/transformers/models/blip_2/convert_blip_2_original_to_pytorch.py

```diff
@@ -147,9 +150,11 @@ def convert_blip2_checkpoint(model_name, pytorch_dump_folder_path=None, push_to_
 
     # load original model
     print("Loading original model...")
-    device = "cuda" if torch.cuda.is_available() else "cpu"
+    hf_model_device = "cuda:1" if torch.cuda.is_available() else "cpu"
+    lavis_device = "cuda:2" if torch.cuda.is_available() else "cpu"
```

Comment:
This is pretty flaky as it assumes there's at least 3 GPU available in the environment